### PR TITLE
added documentation to the welcome_page class

### PR DESCRIPTION
**Pull request by ajwann**
I realize it's probably not the most critical class in the codebase, but I've documented the welcome page class. I'm still on the hunt for other classes that need documentation too.

### CODE REVIEW

**Comment by paulcsmith** [comment]
You can fix the build error by running: `crystal tool format src spec`. That will ensure everything adheres to the official Crystal style. Thanks so much for tackling this. Ping me when you're ready for me to take another look :)

File: src/lucky/welcome_page.cr [hunk 1]
```diff
@@ -74,10 +78,16 @@ class Lucky::WelcomePage
...
+  # Returns the gitter url as a string
+  #
+  #```
+  # gitter_url #=> String
+  #```
   def gitter_url
     "https://gitter.im/luckyframework/Lobby"
   end
 
+  # Returns the CSS styling of the Welcome Page as a string
```
[the remark this answers is not in the document]
Comment (ajwann) [reply src/lucky/welcome_page.cr]: Sure can

**Comment by ajwann** [comment]
@paulcsmith I think we are good to go, just let me know if I need to make any changes!

**Comment by paulcsmith** [comment]
Looks perfect 👌 thanks!

**Pull request merged**